R/coord-funs.R

```diff
@@ -304,3 +324,23 @@ coord_pol_centroids <- function(x, lat = NULL, lon = NULL, drop = TRUE) {
 #   coordinates(wc) <- ~long + lat
 #   wc
 # }
+
+#' @export
+#' @rdname coords
+coord_uncertain <- function(x, coorduncertainityLimit = 30000, drop = TRUE, ignore.na = FALSE){
+  if(!("coordinateUncertaintyInMeters"  %in%  names(x))){
```

Comment:
I couldn't quite understand what you meant exactly. 

Do you intend to use `coord_uncertain` when the `coorduncertainityLimit` doesn't exist?
i.e `dframe(sample_data_1) %>% coord_uncertain()` wouldnt give an error and the output would be `sample_data_1` itself?